query/workmanager_test.go

```diff
@@ -384,7 +386,10 @@ func TestWorkManagerWorkRankingScheduling(t *testing.T) {
 	const numQueries = 4
 	const numWorkers = 8
 
-	wm, workers := startWorkManager(t, numWorkers)
+	workMgr, workers := startWorkManager(t, numWorkers)
+
+	wm, ok := workMgr.(*workManager)
```

Comment:
nit: could use `require.IsType(t, &workManager{}, workMgr)` for a nicer failure message if this ever is `false`. Also, there's a (pre-existing) typo in the godoc comment of this test function.